tests/test_processors.py

```diff
@@ -456,6 +456,17 @@ def test_obtains_exc_info_on_True(self):
         else:
             pytest.fail("Exception not raised.")
 
+    def test_obtains_exc_info_on_number(self):
```

Comment:
I have refactored the tests a bit. Currently it tests for True, int and float. I am not sure if there is a reason to test it with dicts, lists, objects etc. even the float looks like a pretty weird case to me.